tools/src/main/scala/scala/scalanative/codegen/llvm/MetadataCodeGen.scala

```diff
@@ -68,9 +71,24 @@ trait MetadataCodeGen { self: AbstractCodeGen =>
       producer = Constants.PRODUCER,
       isOptimized = attrs.opt == nir.Attr.DidOpt
     )
+    // unmangle method name
+    // see: https://scala-native.org/en/stable/contrib/mangling.html
+    val linkageName = mangled(name)
+    val defnName = if (linkageName.startsWith("_S")) linkageName.drop(2) else linkageName
+    val unmangledName = if (defnName.startsWith("M")) { // subprogram should be a member
+      Unmangle.unmangleGlobal(defnName) match {
+        case Member(owner, sig) =>
+          Unmangle.unmangleSig(sig.mangle) match {
+            case Method(id, _, _) => Some(id)
```

Comment:
Wouldn't use just a method name would be ambiguous?  Let's say we have classes `A` and `B` both defining method `foo`. Then we would trigger break point in each of them. Probably we should use a fully qualified name. `A.foo` or `B.foo`
```suggestion
            case Method(id, _, _) => Some(s"{$owner.id}.${id}")
```